.github/workflows/publish-cli-bins.yaml

```
@@ -9,57 +9,84 @@
-        os: [linux, darwin]
-        arch: [amd64, arm64]
-    env:
-      NAME: flightctl-${{ matrix.os }}-${{ matrix.arch }}
-    runs-on: "ubuntu-24.04"
+        os_arch: [ "linux/amd64", "linux/arm64", "linux/386", "darwin/amd64", "darwin/arm64" ]
+    runs-on: "ubuntu-latest"
+    needs: setup
     steps:
+      - name: Set up Environment Variables
+        id: env_setup
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2129:style:7:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)